diskimage-create/elements/nfp-reference-configurator/post-install.d/1-pkgs

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+apt-get install -y --force-yes python-pip
```

Comment:
Since we dont have ubuntu VM's root credentials
We will start testing without sudo for now.